libsawtooth/Cargo.toml

```diff
@@ -69,13 +70,13 @@ experimental = [
     "redis-store",
     "sqlite",
     "stores",
-    "transaction-receipt-store-lmdb",
     "validator-internals",
 ]
 
 btree-store = ["stores"]
 client = []
 client-rest = ["base64", "client", "reqwest", "serde"]
+lmdb = ["lmdb-zero", "transaction-receipt-store"]
```

Comment:
This should be a cross-cutting feature - it shouldn't depend on the `"transaction-receipt-store"` feature

Reply:
Updated to remove "transaction-receipt-store" dependency